dnf.spec

```diff
@@ -82,7 +82,7 @@ It supports RPMs, modules and comps groups & environments.
 
 Name:           dnf
 Version:        4.2.21
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Please could you keep it with 1. The release we modify only for downstream

Reply:
Sure, easy enough.